validator/testcases/javascript/predefinedentities.py

```diff
@@ -528,6 +577,9 @@ def registry_key(write=False):
                 "method should always return its third argument in cases "
                 "when it is not supplying a specific proxy."),
             "editors_only": True,
+            "signing_help": "Due to the potential for unintended effects, "
+                            "any add-on which uses this API must undergo "
+                            "at least one round of light manual review.",
```

Comment:
`light`? Do we have light manual reviews?Are you referring to preliminary review here? Not sure it's relevant to developers, as they don't explicitly choose their review queue (it only depends on the side-loading or not of their add-ons).